stages/org.osbuild.coreos.squashfs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+import os
+import glob
+import subprocess
+import sys
+
+import osbuild.api
+
+
+def main(tree, mounts, devices):
+    # TODO Fail/Warn if metal is not here?
+    if  devices['metal']:
+        tmp_squashfs_dir = mounts['root']['path']
+        tmpinitrd_rootfs = os.path.join(tree, 'initrd-rootfs')
+        os.makedirs(tmpinitrd_rootfs, exist_ok=True)
```

Comment:
I'll probably figure this out after looking at later stages, but... what is this used for?

If I can't tell, then maybe it's worth adding a comment?